File: 백준/Silver/1929. 소수 구하기/소수 구하기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int M = sc.nextInt();
        int N = sc.nextInt();
        sc.close();

        StringBuilder primeNums = new StringBuilder();
        M = M == 1 ? 2 : M;
        for (int i = M; i <= N; i++) {
            boolean check = true;
            for (int j = 2; j <= Math.sqrt((double) i); j++) {
                if (i % j == 0) {
                    check = false;
                    break;
                }
            }
            if (check) {
                primeNums.append(i).append("\n");
            }
        }
        System.out.print(primeNums);
    }
}

File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
import math

def primeNum(M, N):
    primeNums = list()
    for i in range(M, N+1):
        check = True
        for j in range(2, int(math.sqrt(i)) + 1):
            if i % j == 0:
                check = False
                break
        if check:
            primeNums.append(i)
    return primeNums

M, N = map(int, input().split())
if M == 1:
    M = 2
primeNums = primeNum(M, N)
answer = ""
for prime in primeNums:
    answer += str(prime) + "\n"

print(answer)